src/javascript-node-mongo/devcontainer-template.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "javascript-node-mongo",
-    "version": "3.1.0",
+    "version": "3.1.1",
```

Comment:
Let's bump the `minor` version as we are updating the default

Reply:
done